packages/extension-ui/src/components/Address.tsx

```diff
@@ -72,7 +72,7 @@ function findAccountByAddress(accounts: AccountJson[], _address: string): Accoun
 }
 
 // recodes an supplied address using the prefix/genesisHash, include the actual saved account & chain
-function recodeAddress(
+export function recodeAddress(
```

Comment:
Would be better to do some common file with helper functions right this rather than exporting something from address to use in different settings :)